### PR TITLE
nwc: add hessian file; minor testing updates

### PR DESCRIPTION
**Pull request by vivacebelles**
## Description
Pulling in the hessian file for NWChem jobs. Clumped in with additional testing on QCDB for properties

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [x ] Added hessian file to keep when running via NWChem

## Status
- [x] Changelog updated
- [ x] Ready to go

### CODE REVIEW

File: qcengine/programs/nwchem/harvester.py [hunk 1]
```diff
@@ -198,25 +198,42 @@ def harvest_outfile_pass(outtext):
...
+                    r'^\s+' + r'Total' + r'\s+' + NUMBER + r'\s+' + NUMBER + r'\s*$',
+                    outtext, re.MULTILINE)
+
+            if mobj2:
+                #mbpt_plain = cc_name.replace('\\', '').replace('MBPT', 'MP').replace('(', '').replace(')', '')
+                #print(f'matched tce {mbpt_plain} dipole moment')
+                #only pulling Debye
+                psivar[f'{mbpt_plain} DIPOLE X'] = mobj2.group(2)
+                psivar[f'{mbpt_plain} DIPOLE Y'] = mobj2.group(4)
+                psivar[f'{mbpt_plain} DIPOLE Z'] = mobj2.group(6)
```
Comment (loriab) [inline qcengine/programs/nwchem/harvester.py]: Debye is consistent for now -- will switch to a.u. later.

File: qcengine/programs/nwchem/germinate.py [hunk 2]
```diff
@@ -61,6 +61,9 @@ def muster_modelchem(method: str, derint: int, use_tce: bool) -> Tuple[str, Dict
             mdccmd = f"task tce {runtyp}\n\n"
             opts["tce__ccsd(t)"] = True
         else:
-            mdccmd = f"task ccsd(t) {runtyp}\n\n"
+            mdccmd = f'task ccsd(t) {runtyp}\n\n'
```
Comment (loriab) [inline qcengine/programs/nwchem/germinate.py]: just a note that they'll probably roll out `black` which is a different formatter from `yapf`, and it changes all your `'` to `"`.

**Comment by dgasmith** [comment]
Can you rebase and run black over this?

**Comment by vivacebelles** [comment]
Rebased and formatted with `black` completed.

**Pull request merged**